packages/bot-web-ui/src/components/dashboard/joyride-config.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+import { Text } from '@deriv/components';
+
+type TJoyrideConfig = Record<'showProgress' | 'showSkipButton' | 'spotlightClicks' | 'disableBeacon', boolean>;
+
+type TStep = {
+    label: string | boolean;
+    content: string;
+};
+
+const Step = ({ label, content }: TStep) => (
+    <div className='db-tour'>
+        <Text size='s' line_height='s' color='prominent' as='h' weight='bold'>
+            {label}
+        </Text>
+        <Text size='s' line_height='s' color='prominent' as='p'>
+            {content}
+        </Text>
+    </div>
+);
+
+/**
+ * Joyride specifc config
+ * It should be in camel casing.
+ */
+const joyride_config: TJoyrideConfig = {
+    showProgress: true,
+    showSkipButton: true,
+    spotlightClicks: true,
+    disableBeacon: false,
+};
+
+export const DBOT_ONBOARDING = [
```

Comment:
```suggestion
export const dbot_onboarding = [
```
looks like a regular array as well 🙂